### PR TITLE
Lpa 2130 confirm message

### PR DESCRIPTION
**Pull request by Biggsen**

### CODE REVIEW

File: module/Application/src/Application/Controller/Authenticated/DashboardController.php [hunk 1]
```diff
@@ -160,6 +160,27 @@ public function deleteLpaAction()
...
+
+        $viewModel = new ViewModel([
+            'lpaId' => $lpa->id,
+            'donorName' => $lpa->document->donor->name
+        ]);
+
+        $viewModel->setTemplate('application/dashboard/confirm-delete.twig');
+
+        if ($this->getRequest()->isXmlHttpRequest()) {
+            //$viewModel->setTerminal(true);
```
Comment (taylorsj1980) [inline module/Application/src/Application/Controller/Authenticated/DashboardController.php]: Please remove comment or line. Is this required?

File: module/Application/src/Application/Controller/Authenticated/Lpa/PeopleToNotifyController.php [hunk 2]
```diff
@@ -164,6 +165,40 @@ public function editAction()
         return $viewModel;
     }
 
+        public function confirmDeleteAction()
```
Comment (taylorsj1980) [inline module/Application/src/Application/Controller/Authenticated/Lpa/PeopleToNotifyController.php]: Small alignment fix

File: module/Application/src/Application/Controller/Authenticated/Lpa/PrimaryAttorneyController.php [hunk 3]
```diff
@@ -203,6 +204,47 @@ public function editAction()
...
+            $attorney = $lpaDocument->primaryAttorneys[$attorneyIdx];
+        }
+
+        // if attorney idx does not exist in lpa, return 404.
+        if (!isset($attorney)) {
+            return $this->notFoundAction();
+        }
+
+        // Setting the trust flag
+        if (isset($attorney->number)) {
```
Comment (taylorsj1980) [inline module/Application/src/Application/Controller/Authenticated/Lpa/PrimaryAttorneyController.php]: Please condense this if statement to...
`$isTrust = isset($attorney->number);`

File: module/Application/src/Application/Controller/Authenticated/Lpa/ReplacementAttorneyController.php [hunk 4]
```diff
@@ -194,6 +195,39 @@ public function editAction()
...
+
+        if (array_key_exists($attorneyIdx, $lpaDocument->replacementAttorneys)) {
+            $attorney = $lpaDocument->replacementAttorneys[$attorneyIdx];
+        }
+
+        // if attorney idx does not exist in lpa, return 404.
+        if (!isset($attorney)) {
+            return $this->notFoundAction();
+        }
+
```
Comment (taylorsj1980) [inline module/Application/src/Application/Controller/Authenticated/Lpa/ReplacementAttorneyController.php]: You need to add the `$isTrust` logic (from action of the same name in PrimaryAttorneyController) into this controller also.
A trust can be used as a replacement attorney if one hasn't been used elsewhere.

File: module/Application/src/Application/Controller/Authenticated/DashboardController.php (hunk 1 above)
Comment (Biggsen) [inline module/Application/src/Application/Controller/Authenticated/DashboardController.php]: uncommenting.

File: module/Application/src/Application/Controller/Authenticated/Lpa/PeopleToNotifyController.php (hunk 2 above)
Comment (Biggsen) [inline module/Application/src/Application/Controller/Authenticated/Lpa/PeopleToNotifyController.php]: aligned.

File: module/Application/src/Application/Controller/Authenticated/Lpa/PrimaryAttorneyController.php (hunk 3 above)
Comment (Biggsen) [inline module/Application/src/Application/Controller/Authenticated/Lpa/PrimaryAttorneyController.php]: nice

File: module/Application/src/Application/Controller/Authenticated/Lpa/ReplacementAttorneyController.php (hunk 4 above)
Comment (Biggsen) [inline module/Application/src/Application/Controller/Authenticated/Lpa/ReplacementAttorneyController.php]: ok. done.

**Pull request merged**